README.md

```diff
@@ -1,3 +1,19 @@
 # Rock Paper Scissors - Git
 
-Some description.
+We are learning **git** & **GitHub**.
+
+`hello from vim`
+
+> :wq
+
+## Topics:
```

Comment:
This list could be converted to a table with checkmarks.
This way we could track the progress of our study process.